### PR TITLE
updated installation procedure

### PR DESCRIPTION
**Pull request by ebrensi**
Two things here: 
- I updated `README.md` to include some info about setting up with Anaconda Python distribution, which uses it's own environment system (although virtualenv still works), and some more detailed instructions about MySQL setup.
- I took branch specifications out of `requirements.txt` so it just installs the latest versions of `django-calaccess-raw-data` and `swagger-py`. I hope that's not a bad thing.  It seems to work for me.  Feel free to change that before you pull.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
--e git://github.com/caciviclab/django-calaccess-raw-data.git@0dd6b6913e4a00b49d1b9c9bf9829ed83f4587a3#egg=django-calaccess-raw-data
+-e git://github.com/caciviclab/django-calaccess-raw-data.git#egg=django-calaccess-raw-data
```
Comment (tdooner) [inline requirements.txt]: As I mentioned in #71 , it is important that we keep the `@0dd6b6913e4a00b49d1b9c9bf9829ed83f4587a3` version pin here.

File: requirements.txt [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
 mysqlclient
 Pillow==3.0.0
--e git+https://github.com/adborden/swagger-py.git@dab635c6c242b11c81add0069c7d9ee1c842e866#egg=swaggerpy-master
```
Comment (tdooner) [inline requirements.txt:4]: And here too.

**Comment by ebrensi** [comment]
I understand why we won't pull changes to requirements.txt, but how about the updated README?  I think it's valid.

File: README.md [hunk 3]
```diff
@@ -56,17 +62,42 @@ mysql> create database calaccess_raw;
 mysql> \q
 python disclosure-backend/manage.py migrate
 ```
+(If mysql exits with a complaint about a password, use `mysql -p --user root`)
+
+
+#### Modify `settings.py` (or create `settings_local.py`)
+In `disclosure-backend\project\settings.py` you'll find the database specification
```
Comment (tdooner) [inline README.md]: These should be forward-slashes for unix OSes

**Comment by tdooner** [comment]
:tada: Thanks!

**Pull request merged**